aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -1340,7 +1340,7 @@ The preceding code calls the method with a null product if no request body is se
 | --------- | -------------- |
 | `/products?pageNumber=3` | `3` returned |
 | `/products` | `1` returned |
-| `/products/two` | `BadHttpRequestException`: Failed to bind parameter `"Nullable<int> pageNumber"` from "two".|
+| `/products/two` | HTTP 404 error, no matching route |
```

Comment:
@collinstevens why doesn't the route match? We should add this row and update the original row to something the throws. Maybe `/products/0x2` or  `/products/2x`

Reply:
I agree with @Rick-Anderson, we could change the current row with `/products?pageNumber=two` and add this new line.

Reply:
@Rick-Anderson I believe the route `/products/two` returns a `404` because there is no route with a configured route template of `/products/{*path}` or `/products/two/`; not because of model binding as the original documentation would indicate.

Here is a repository to use for this example. https://github.com/collinstevens/MinimalApiProductRoute

I'm not even sure if the documentation _should_ include something which would throw because the header of the example is `If invalid data is provided and the parameter is nullable, the route handler is not run.`, therefore, I would expect model binding to **never** have run to begin with, and not to see the model binding exception.

Nevertheless, @brunolins16 and @Rick-Anderson, I've updated the branch to include the suggested new line and the current line I modified. I also added the changes to the previous framework moniker I originally missed.